_docs-sources/pipelines/overview/configuration.md

```diff
@@ -0,0 +1,88 @@
+
+A sample `config.yml` with a full set of options is below.
+
+```
+pipelines:
+  # Branch that IAC is deployed from
+  deploy-branch-name: main
+  # Version of the https://github.com/gruntwork-io/terraform-aws-security modules used in templates
+  module-security-version: v0.68.3
+  # Version of the https://github.com/gruntwork-io/terraform-aws-control-tower modules used in templates
+  control-tower-modules-version: v0.4.2
+  # Version of the https://github.com/gruntwork-io/terraform-aws-architecture-catalog modules used in templates
+  arch-catalog-version: v2.2.5
+  # (Optional) Allow override the repository in use for the architecture catalog
+  arch-catalog-repo-name: gruntwork-io/terraform-aws-architecture-catalog
+  # (Optional) Allow overriding the base path within the architecture catalog
+  arch-catalog-base-path: ./terraform-aws-architecture-catalog
+  # (Optional) Override the path within the architecture catalog where to find the template for boilerplating new account baselines
+  single-account-baseline-template-path: /templates/single-account-baseline
+  # Default AWS region for infrastructure managed in this repository
+  default-aws-region: us-east-1
+  # GitHub Organization this repository belongs to
+  github-org: gruntwork-io-demo
+  # Name of the infrastructure-live-access-control repository
+  access-control-repo-name: sales-demo-infra-live-access-control
+  # Name of the infrastructure-modules repository
+  infra-modules-repo-name: infrastructure-modules
+  # Version of the https://github.com/gruntwork-io-demo/infrastructure-modules modules used in templates
+  infra-modules-release-version: v0.1.0
+  # The IaC runtime binary, valid options are opentofu or terraform
+  tf-binary: opentofu
+  # (Optional) override the folder for the security account, default is 'security'
+  security-account-name: security
+  # (Optional) override the folder for the shared account, default is 'shared'
+  shared-account-name: shared
+  # (Optional) override the folder for the logs account, default is 'logs'
+  logs-account-name: logs
+  # (Optional) override the folder for the management account, default is 'management'
+  management-account-name: management
+
+
+  ###########################################################################
+  # Configuration options below are ONLY for Pipelines Enterprise customers
+  ###########################################################################
+
+  # Only set to true for delegated (vended) accounts from account factory
+  is-delegated-repo: false
+  # Only set to true for infra-live-access-control
+  is-access-control-repo: false
+
+  account-vending:
+    sdlc:
+    # Map of account identifiers to account vending configurations.
+    # A valid account vending configuration includes:
+    # - account-identifiers: List of account identifiers that will be created (required).
```

Comment:
I think it's confusing to have what looks like valid YAML list elements under `sdlc` in this comment. Consider placing each comment right above the relevant configuration.